plugin/cog/pom.xml

```diff
@@ -92,7 +93,47 @@
                 <artifactId>ehcache</artifactId>
                 <version>${ehcache.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.azure</groupId>
+                <artifactId>azure-storage-blob</artifactId>
+                <version>${azure.version}</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
+    
+    <profiles>
+        <profile>
+            <!--
+             Converges the netty dependency version from cog-rangereader-s3 and cog-rangereader-azure
+             to a common version that works with both, so that both plugins can co-exist without
+             adding netty libs with slightly different versions to the dependencies.
+
```

Comment:
Ok but... in time they might diverge enough that a binary incompatibility arises.
I don't see a production use case where both plugins need to be installed at the same time, one either deploys on AWS or on Azure (remote access/testing might be an option but performance would be so bad that's not usable beyond  simple tests).

Reply:
In GeoServer-Cloud, since the docker images are built with all integrated plugins, I need them both to be installed, and I have to do this dependency convergence anyway.
I think it'd still be a good idea to make sure both are compatible here anyways? there's the online tests to make sure they work.

Reply:
I'm ambivalent, but let's keep as you propose and see if/when it breaks.

Reply:
great, thanks